src/dtls.c

```diff
@@ -107,14 +107,14 @@ int DtlsIgnoreError(int err)
 {
     /* Whitelist of errors not to ignore */
     switch (err) {
-    case MEMORY_E:
-    case MEMORY_ERROR:
-    case ASYNC_INIT_E:
-    case ASYNC_OP_E:
-    case SOCKET_ERROR_E:
-    case WANT_READ:
-    case WANT_WRITE:
-    case COOKIE_ERROR:
+    case WC_NO_ERR_TRACE(MEMORY_E):
```

Comment:
This doesn't look right either!

Reply:
Nothing wrong there.  The `switch()` cases in `DtlsIgnoreError()`, like all the other similar `switch()`es elsewhere, are uniformly annotated with `WC_NO_ERR_TRACE()`.  This is actually _required_, rather than just desired, because C requires that `case`s evaluate to integer constants.